### ISSUE TITLE
The results on nuscenes validation set .

### ISSUE THREAD

**Issue by Pedrojj97** [yanx27/2DPASS#15]
Thanks for you great work, I test the checkpoint you released on the nuscenes validation set, but it only got only 73.7% mIoU, which is much lower than you reported。 I wonder if you have any suggestions on this。

**Comment by yanx27** [comment]
@Pedrojj97 Could you give me more details about your environment or how to test the model?

**Comment by Pedrojj97** [comment]
thanks for your reply.
I run the test by python main.py--config config/2DPASS-nuscenese.yaml --gpu 0 --test --num_vote 1 --checkpoint ./checkpoints/best_model.ckpt.
The envs are listed as follow:
python 3.6.5
pytorch 1.5.0+cu10.1
pytorch-lightning 1.3.8
torchmetrics 0.5
torch-scatter 2.0.4

**Comment by yanx27** [comment]
> thanks for your reply. I run the test by python main.py--config config/2DPASS-nuscenese.yaml --gpu 0 --test --num_vote 1 --checkpoint ./checkpoints/best_model.ckpt. The envs are listed as follow: python 3.6.5 pytorch 1.5.0+cu10.1 pytorch-lightning 1.3.8 torchmetrics 0.5 torch-scatter 2.0.4

Please refer to the README and use PyTorch > 1.8.

**Comment by Pedrojj97** [comment]
> > thanks for your reply. I run the test by python main.py--config config/2DPASS-nuscenese.yaml --gpu 0 --test --num_vote 1 --checkpoint ./checkpoints/best_model.ckpt. The envs are listed as follow: python 3.6.5 pytorch 1.5.0+cu10.1 pytorch-lightning 1.3.8 torchmetrics 0.5 torch-scatter 2.0.4
> 
> Please refer to the README and use PyTorch > 1.8.

Thanks, I will try with the pytorch>1.8.

**Comment by yanx27** [comment]
Hi @Pedrojj97 , I re-test the released checkpoint, and it seems to work well.  If you meet other issue, please let me know.
![ea94d82688c713332e55dbe9bcd8db2](https://user-images.githubusercontent.com/35829331/194971953-05312789-738f-4f2d-8f82-7b7f31c92623.jpg)

**Comment by PinTang1999** [comment]
> Hi @Pedrojj97 , I re-test the released checkpoint, and it seems to work well. If you meet other issue, please let me know. ![ea94d82688c713332e55dbe9bcd8db2](https://user-images.githubusercontent.com/35829331/194971953-05312789-738f-4f2d-8f82-7b7f31c92623.jpg)

Hi,  @yanx27  @Pedrojj97 Could you tell me how many vote num did you set on nuscenes val set to get this result?

**Comment by wangcy0802** [comment]
> > 你好@Pedrojj97，我重新测试了发布的检查点，看起来效果很好。如果您遇到其他问题，请告诉我。![ea94d82688c713332e55dbe9bcd8db2](https://user-images.githubusercontent.com/35829331/194971953-05312789-738f-4f2d-8f82-7b7f31c92623.jpg)
> 
> 测试结果与你用pytorch==1.10报告的结果相同，再次感谢你的回复。

我有个问题，为什么打印的是这些类别的指标，不应该是lable_16里的那些类别吗

**Comment by wangcy0802** [comment]
> > Hi @Pedrojj97 , I re-test the released checkpoint, and it seems to work well. If you meet other issue, please let me know. ![ea94d82688c713332e55dbe9bcd8db2](https://user-images.githubusercontent.com/35829331/194971953-05312789-738f-4f2d-8f82-7b7f31c92623.jpg)
> 
> Hi, @yanx27 @Pedrojj97 Could you tell me how many vote num did you set on nuscenes val set to get this result?

为什么显示的是这些类别而不是lable_16里的那些类别呢
